tests/Common/ComponentsTest.php

```diff
@@ -1162,15 +1152,20 @@ public function testDuplicateConfigShouldNotBeCreated()
 
     public function testPermissions()
     {
-        $tokenId = $this->_client->createToken(array(), 'test');
+        $tokenOptions = (new TokenCreateOptions())
+            ->setDescription('test')
+        ;
+
+        $tokenId = $this->_client->createToken($tokenOptions);
         $token = $this->_client->getToken($tokenId);
 
-        $client = new \Keboola\StorageApi\Client(array(
+        $client = new \Keboola\StorageApi\Client([
```

Comment:
zbytečný namespace

Reply:
Zbytecny "neni" protoze `Client` v `use` neni.
To mam celou classu zrefaktorovat a vsehny classy s namespacem pridat do `use`? Je tam toho bambilion :-)